### PR TITLE
[sonic_installer]: add command to cleanup installed images

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: sonic_installer/main.py [hunk 1]
```diff
@@ -291,5 +295,24 @@ def binary_version(binary_image_path):
...
+        expose_value=False, prompt='Cleanup images which are not current and next, continue?')
+def cleanup():
+    """ Cleanup installed images which are not current and next """
+    images = get_installed_images()
+    curimage = get_current_image()
+    nextimage = get_next_image()
+    image_removed = 0
+    for image in get_installed_images():
+        if image != curimage and image != nextimage:
+            click.echo("Cleanup image %s" % image)
```
Comment (jleveque) [inline sonic_installer/main.py]: Suggest to change from `Cleanup image %s` to `Removing image %s`

File: sonic_installer/main.py [hunk 2]
```diff
@@ -291,5 +295,24 @@ def binary_version(binary_image_path):
...
+            remove_image(image)
+            image_removed += 1
+
+    if image_removed == 0:
+        click.echo("No image to cleanup")
```
Comment (jleveque) [inline sonic_installer/main.py]: Suggest to change from `No image to cleanup` to `No image(s) to remove`

File: sonic_installer/main.py (hunk 1 above)
Comment (lguohan) [reply sonic_installer/main.py]: done

File: sonic_installer/main.py (hunk 2 above)
Comment (lguohan) [reply sonic_installer/main.py]: done

**Pull request merged**